services/auth-machine/README.md

```diff
@@ -0,0 +1,17 @@
+# Auth-Machine
+
+The oCIS Auth Machine is used for interservice communication when using user impersonation.
+
+ocis uses serveral authentication services for different use cases. All services that start with `auth-` are part of the authentication service family. Each member authenticates requests with different scopes. As of now these services exist:
+  -   `auth-basic` handles basic authentication
+  -   `auth-bearer` handles oidc authentication
+  -   `auth-machine` handles interservice authentication when a user is impersonated
+  -   `auth-service` handles interservice authentication when using service accounts
+
+## User Impersonation
+
+When one ocis service is trying to talk to another ocis services it needs to authenticate itself. To do so it will impersonate a user using the `auth-machine` service. It will then act on behalf on this users. Any action will show up (e.g. in the audit log) as actions of this specific user.
```

Comment:
```suggestion
When one ocis service is trying to talk to other ocis services, it needs to authenticate itself. To do so, it will impersonate a user using the `auth-machine` service. It will then act on behalf of this user. Any action will show up as action of this specific user, visible when e.g. logged in the audit log .
```